### PR TITLE
qt515: update qtscript version

### PR DESCRIPTION
**Pull request by tjni**
###### Description of changes

I hope this fixes issues like [this one](https://hydra.nixos.org/build/193898947/nixlog/1). Related to https://github.com/NixOS/nixpkgs/pull/193494.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by tjni** [comment]
Setting as a draft until `ofborg` results come back.

**Comment by tjni** [comment]
@ofborg build qt515

**Comment by tjni** [comment]
@ofborg build CuboCore

**Comment by LunNova** [comment]
Introduced in https://github.com/NixOS/nixpkgs/commit/dd4afc7c0055523096f64ff941128d040dcc91ac

File: pkgs/development/libraries/qt-5/5.15/srcs.nix [hunk 1]
```diff
@@ -3,7 +3,7 @@
 let
   version = "5.15.6";
   overrides = {
-    qtscript.version = "5.15.4";
+    qtscript.version = "5.15.6";
```
Comment (LunNova) [inline pkgs/development/libraries/qt-5/5.15/srcs.nix:6]: This is equivalent to removing the override since it matches.

**Comment by LunNova** [comment]
cc @K900 since you added it

**Comment by tjni** [comment]
@ofborg build CuboCore

File: pkgs/development/libraries/qt-5/5.15/srcs.nix (hunk 1 above)
Comment (tjni) [reply pkgs/development/libraries/qt-5/5.15/srcs.nix]: Good catch, thanks!

**Comment by K900** [comment]
Ugh. I forgot those existed, again. Should be good.

**Comment by vcunat** [comment]
Ah, I'm sorry, I wasn't looking at PRs.  I didn't expect you'd be so quick, so I just pushed the same as 7cce28d59.

**Pull request closed without merging**